### DISCUSSION TITLE
New setup

### DISCUSSION THREAD

**Discussion by Shand359** [aaronwmorris/indi-allsky#678] [Q&A] [answered]
I hope someone can help with this issue. 

I have INDI-Allsky installed on my RPi4 8Gb, but im having issues with the web GUI. It keeps coming up as "not secure" and occasionally will not even load in at all....

**Comment by aaronwmorris** [comment, marked as answer]
The web interface is using a self-signed certificate, so the browser is going to report it as "not secure".
